app/cli/cmd/root.go

```diff
@@ -83,6 +93,38 @@ func NewRootCmd(l zerolog.Logger) *cobra.Command {
 			return nil
 		},
 		PersistentPostRunE: func(cmd *cobra.Command, args []string) error {
+			// Do not track telemetry if the env variable is set
+			var telemetryClient telemetry.Client
+			if os.Getenv(doNotTrackEnv) == "1" || os.Getenv(doNotTrackEnv) == "true" {
```

Comment:
it would be nice if the detection could be extracted as a function `isTelemetryDisabled() bool`